content/en/docs/setup/install/operator/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Operator Install [Experimental]
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation is a shorter process with the option of
+installing Istio using a one-line command. It has user input
+validation to help prevent installation errors and customization options to
+override any aspect of the configuration.
+
+It has the following additional characteristics:
+
+- Descriptive error messages to spot configuration errors easier.
+
+- Installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [`istioctl`](/docs/reference/commands/istioctl/)
+commands and a new Operator API.
```

Comment:
I think this operator API is in contention.  The last state of things was that the TOC was going to offer some guidance based upon the environment wg's endpoint objectives.  It will take some time to produce the endpoint documentation (@costinm can comment if he likes).  I'd just delete the comment `and a new Operator API.`

Reply:
addressed